pwa/app/lib/utils.ts

```diff
@@ -9,6 +11,25 @@ export function removeNonNumeric(str: string): string {
   return str.replace(/\D/g, '');
 }
 
+export async function parseParamsForYearElseDefault(
```

Comment:
We don't need to do this in this PR, but we could create a provider component for "current year" and wrap the dom tree in it, and that way we can simply call a hook like `useCurrentSeason()` or the like

Reply:
I don't think that works because this is called during SSR?